### PR TITLE
convert double backticks to single backtick

### PR DESCRIPTION
**Pull request by ringabout**
Ref https://github.com/nim-lang/Nim/pull/16994#issuecomment-778509436

### CODE REVIEW

File: lib/packages/docutils/rstast.nim [hunk 1]
```diff
@@ -333,7 +333,7 @@ proc renderRstToJson*(node: PRstNode): string =
 proc renderRstToStr*(node: PRstNode, indent=0): string =
   ## Writes the parsed RST `node` into a compact string
   ## representation in the format (one line per every sub-node):
-  ## ``indent - kind - text - level - order - anchor (if non-zero)``
+  ## `indent - kind - text - level (if non-zero)`
```
Comment (konsumlamm) [inline lib/packages/docutils/rstast.nim]: Is it intentional that you removed `- order - anchor `?

File: lib/pure/os.nim [hunk 2]
```diff
@@ -2288,8 +2288,8 @@ iterator walkDirRec*(dir: string,
   ## ---------------------   ---------------------------------------------
   ## followFilter            meaning
   ## ---------------------   ---------------------------------------------
-  ## ``pcDir``               follow real directories (default)
-  ## ``pcLinkToDir``         follow symbolic links to directories
+  ## `pcDir`               follow real directories (default)
+  ## `pcLinkToDir`         follow symbolic links to directories
```
Comment (konsumlamm) [inline lib/pure/os.nim]: You may want to fix the indentation here.

File: lib/pure/parseopt.nim [hunk 3]
```diff
@@ -75,27 +75,27 @@
 ## The `getopt iterator<#getopt.i,OptParser>`_, which is provided for
 ## convenience, can be used to iterate through all command line options as well.
 ##
-## ``shortNoVal`` and ``longNoVal``
+## `shortNoVal` and `longNoVal`
 ## ================================
```
Comment (konsumlamm) [inline lib/pure/parseopt.nim]: Nit: You may want to adjust the amount of `=`s here.

Comment (ringabout) [reply lib/pure/parseopt.nim]: thanks

File: lib/pure/os.nim (hunk 2 above)
Comment (ringabout) [reply lib/pure/os.nim]: it has bugs when single backtick in markdown tables

File: lib/packages/docutils/rstast.nim (hunk 1 above)
Comment (ringabout) [reply lib/packages/docutils/rstast.nim]: thanks

File: lib/pure/os.nim [hunk 4]
```diff
@@ -2321,8 +2321,8 @@ iterator walkDirRec*(dir: string,
...
```
Comment (konsumlamm) [inline lib/pure/os.nim]:
```suggestion
  ## `pcDir`                 follow real directories (default)
  ## `pcLinkToDir`           follow symbolic links to directories
```
Not sure what you're referring to in https://github.com/nim-lang/Nim/pull/17041#discussion_r576486835.

Comment (ringabout) [reply lib/pure/os.nim]: thanks

**Comment by Araq** [comment]
Please split this up into multiple PRs, too risky to merge this way, reviewers can get tired.

**Comment by timotheecour** [comment]
@Araq just to avoid running in circles, how many PR's is appropriate?

I'm asking because of dom's comment in https://github.com/nim-lang/Nim/pull/16994#issuecomment-778492559

> I'd say there should be a single PR made to change them all at once to avoid noise in PRs that just want to improve documentation.

How about the following approach:
send a script (nim, python or bash) that generates this PR, along with the PR; then reviewer can verify the script, and double check they get the same results by running the script on their end, using git diff.

Edge cases (requiring re-formatting as in https://github.com/nim-lang/Nim/pull/17041#discussion_r576340661) could be left out or deferred to future PR.

**Comment by Araq** [comment]
A PR shouldn't touch more than 10 files.

**Pull request closed without merging**